### PR TITLE
Flamer must be fully wielded before firing

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Make it so you have to finish wielding before firing a flamer

## Why It's Good For The Game

There is no innacuracy in flamer, so firing while wielding or after was exactly the same. This made chasing with a flamer too strong for good players, as the xeno cannot really do anything against this loadout, not even flee

## Changelog
:cl:
balance: You cannot shoot the flamer if you didn't finish your wielding
/:cl:

### CODE REVIEW

**Comment by joookschad** [comment]
wack

**Comment by Dolth** [comment]
I approve. But this won't save you. Muhahahahaha!

**Comment by David-Stormwell** [comment]
This had it coming sooner or later

**Comment by RyderFive** [comment]
Am gonna take at least 50% credit for this.

File: code/__DEFINES/conflict.dm [hunk 1]
```diff
@@ -57,24 +57,25 @@
...
+#define GUN_ENERGY						(1<<9)
+#define GUN_FLASHLIGHT_ON				(1<<10)
+#define GUN_WIELDED_FIRING_ONLY			(1<<11)
+#define GUN_POLICE						(1<<12)
+#define GUN_BURST_FIRING				(1<<13)
+#define GUN_ALLOW_SYNTHETIC				(1<<14)
+#define GUN_HAS_AUTOBURST				(1<<15)
+#define GUN_IS_AIMING					(1<<16)
+#define AUTO_AIM_MODE					(1<<17)
+#define GUN_WIELDED_STABLE_FIRING_ONLY	(1<<18)
```
Comment (TiviPlus) [inline code/__DEFINES/conflict.dm:78]: Dont do this just leave one space

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**